nginx/nginx.go

```diff
@@ -307,21 +309,16 @@ func (lb *nginxLoadBalancer) createConfig(update controller.IngressUpdate) ([]by
 func createNginxEntries(update controller.IngressUpdate) []*nginxEntry {
 	sortedIngressEntries := update.SortedByName().Entries
 	indexedEntries := make(map[string]*nginxEntry)
+	hostPaths := make(map[string]map[string]struct{})
```

Comment:
up to you, i think to the seasoned go programmer it makes sense given the number of things i found about using a map as a set